MAUI/AIAssistView/suggestions.md

```diff
@@ -0,0 +1,257 @@
+
+        this.GenerateAssistItems();
+     }
+
+    private async Task GetResult(AssistItem requestItem)
+    {
+        await Task.Delay(1000).ConfigureAwait(true);
+
+        AssistItem responseItem = new AssistItem()
+        {
+           // response from AI service
+           Text = "MAUI stands for .NET Multi-platform App UI. It's a .NET framework for building cross-platform apps with a single C# codebase for iOS, Android, macOS, and Windows. Sure! Here's a link to learn more about .NET MAUI",
+           Suggestion = assistSuggestions,
+        };
+
+        // Add the response item to the collection
+        this.AssistItems.Add(responseItem);
+    }
+
+    ...
+ }
+
+{% endhighlight %}
+{% endtabs %}
+
+### Add an image in suggestion item
+
+In the `SfAIAssistView` control, an image can be added to the suggestion list by assigning an image source to the `ImageSource` property.
+
+{% tabs %}
+{% highlight c# tabtitle="ViewModel.cs" hl_lines="13 14" %}
+   
+ public class SuggestionsViewModel : INotifyPropertyChanged
+ {
+
+    ....
+
+    public SuggestionsViewModel()
+    {
+        this.AssistItems = new ObservableCollection<object>();
+        
+        assistSuggestions = new AssistItemSuggestion();
+
+        suggestions = new ObservableCollection<ISuggestion>();
+        suggestions.Add(new AssistSuggestion() { Text = "Get started with .NET MAUI", ImageSource = "learn_more.png" });
+        suggestions.Add(new AssistSuggestion() { Text = "Build your first MAUI app", ImageSource = "get_started.png" });
+
+        assistSuggestions.Items = suggestions;
+     }
+
+    ...
+
+ }
+
+{% endhighlight %}
+{% endtabs %}
+
+### Change the orientation of suggestions
+
+The `AssistItemSuggestion.Orientation` property allows you to display suggestion items in either horizontal or vertical orientation. The default orientation is vertical.
+
+{% tabs %}
+{% highlight c# tabtitle="ViewModel.cs" hl_lines="13" %}
+
+    public SuggestionsViewModel()
+    {
+        this.AssistItems = new ObservableCollection<object>();
+        
+        assistSuggestions = new AssistItemSuggestion();
+
+        suggestions = new ObservableCollection<ISuggestion>();
+        suggestions.Add(new AssistSuggestion() { Text = "Get started with .NET MAUI" });
+        suggestions.Add(new AssistSuggestion() { Text = "Build your first MAUI app" });
+
+        assistSuggestions.Items = suggestions;
+
+        assistSuggestions.Orientation = SuggestionsOrientation.Horizontal;
+     }
+ 
+ 
+{% endhighlight %}
+{% endtabs %}
+
+
+## Prevent the chosen suggestion from being sent automatically.
+
+By default, when tapping or clicking the suggestion item, it will be sent as a request item immediately. So, to restrict this behavior and show the suggestion list in editor view, you can set the `SuggestionItemSelectedEventArgs.CancelRequest` to `true` within the `SuggestionItemSelected` event handler or the `SuggestionItemSelectedCommand` command's execution.
```

Comment:
By default, when tapping or clicking the suggestion item, it will be sent as a request item immediately. So, to restrict this behavior and show the selected suggestion item in editor view, you can set the `SuggestionItemSelectedEventArgs.CancelRequest` to `true` within the `SuggestionItemSelected` event handler or the `SuggestionItemSelectedCommand` command's execution.

Reply:
Updated the requested change